cost-analyzer/values.yaml

```diff
@@ -2822,6 +2822,12 @@ kubecostAdmissionController:
 costEventsAudit:
   enabled: false
 
+## This flag is only required for users upgrading from 1.x to 2.x. Ensure users
+## are aware of any changes that might need to be made to their architecture
+## before continuing with the upgrade.
+##
+upgradeToKubecostV2: false
```

Comment:
I suggest making this a map so we can repurpose the struct for future major versions if the need should arise (and, let's be honest, it probably will:

```yaml
upgrade:
  fromV1: true
```

Reply:
Great idea, doing that now!

Reply:
@chipzoller What do you think about this modification

```
upgradeKubecost:
  toV2: true
```

Reply:
Because this chart is not called "Kubecost" but rather "Cost Analyzer" I think this creates a small amount of confusion. Should we implement the complimentary changes that have been proposed but not accepted for the [2.0 Helm chart](https://docs.google.com/document/d/17GJkIS2ce5w0u9qmaBJYIi99NTs_dEs6Iid83aOZw2w/) (Kubecost internal document), this works perfectly. Otherwise, I would drop the `Kubecost` portion of the key but that is a minor detail.